elastalert/kibana.py

```diff
@@ -166,6 +166,8 @@
                   u'style': u'dark',
                   u'title': u'ElastAlert Alert Dashboard'}
 
+kibana4_time_temp = "(refreshInterval:(display:Off,section:0,value:0),time:(from:'%s',mode:absolute,to:'%s'))"
```

Comment:
Good point on the set time window. Given that, I think punting auto-refresh makes sense. If we were to ever support more flexible versions of the time windowing, allowing auto-refresh might make more sense.